packages/react-components/react-table/etc/react-table.api.md

```diff
@@ -376,19 +388,21 @@ export const tableHeaderCellClassName = "fui-TableHeaderCell";
 export const tableHeaderCellClassNames: SlotClassNames<TableHeaderCellSlots>;
 
 // @public
-export type TableHeaderCellProps = ComponentProps<Partial<TableHeaderCellSlots>> & {
+export type TableHeaderCellProps = ComponentProps<Partial<TableHeaderCellSlots>> & Pick<ColumnWidthProps, 'resizeHandleMouseDown'> & {
```

Comment:
Please rebuild the API file and/or figure out why `resizeHandleMouseDown` is leaking in public API.